static/css/components/carousel.less

```diff
@@ -113,4 +118,27 @@
     overflow: hidden;
   }
 }
+
+@padding-blank-cover: 15px;
+@height-blank-cover: 200px - (@padding-blank-cover * 2);
+.carousel__item__blankcover {
+  position: relative;
+  padding: @padding-blank-cover;
+  text-align: center;
+  background: @lighter-grey;
+  font-size: 1em;
+
+  > div {
```

Comment:
The readability of these goes straight to 0 without class names; let's name them `title` and `byline`

Reply:
What do you mean by "these" here? Specifically

`@padding-blank-cover` and `@height-blank-cover`?